runtime/doc/autocmd.txt

```diff
@@ -466,6 +466,16 @@ CompleteDone			After Insert mode completion is done.  Either
 				CompleteDonePre if you need it.
 				|v:completed_item| gives the completed item.
 
+				Sets these |v:event| keys:
+				    reason	Reason for completion being
```

Comment:
did we already discuss adding this as a field `v:completed_item` (and `:help complete-items`) ?

Reply:
I don't see how that would work? How can you signal `abort` then without a completion item?

 It also seems semantically inappropriate since the reason belongs to the the completion event, not the item.

Reply:
This has nothing to do with individual completion items, it's about completion in general